### ISSUE TITLE
Update Question

### ISSUE THREAD

**Issue by LegitJesseee** [OlivierCroisier/LP4J#2]
Hi Olivier! I was wondering if you could update this library to work with the new lineup of Launchpads. 
I noticed that if you change the DEVICE_SIGNATURE String in MidiDeviceConfiguration.java, the Launchpad will still respond to certain commands, however it is obvious that much has changed in the newer iterations.

**Comment by OlivierCroisier** [comment]
Hi Jesse !

I just took a look at their new Launchpad Pro, it seems to be a whole different beast, with pressure-sensitive pads with more colors, and some more buttons around the frame. Nice upgrade !

Unfortunately, I don't own one of those new pads (and they are fairly expensive), so it would be difficult for me to adapt LP4J to handle them properly. Also, I must check if Novation published the new protocol - whithout which I can't do much.

I sent a short email to Novation when I first published LP4J to inform them and get feedback, but they never replied. That's too bad, I think they could only benefit from an opensource "driver" for their products...
Also, I'm very curious about what you built (or intend to build) with LP4J ?

Olivier
